File: wwwroot/js/WeatherForecast.js
function refreshWeatherForecast() {
    $(document).ready(function () {
        $.ajax({
            url: '/WeatherForecast',
            success: function (data) {
                let tommorrow = data[0];
                let date = new Date(tommorrow.date).toDateString();
                $('#date').text(date);
                $('#temperature').text(tommorrow.temperatureC + ' C');
                $('#summary').text(tommorrow.summary);
            },
            error: function (data) {
                alert('Failed to load data');
            }
        });
    });
}


setInterval(refreshWeatherForecast, 3000);

File: Models/WeatherForecast.cs
using System;

namespace InternshipClass.WebAPI
{
    public class WeatherForecast
    {
        public DateTime Date { get; set; }

        public int TemperatureC => (int)(TemperatureK - 273.15);

        public string Summary { get; set; }

        public double TemperatureK { get; set; }
    }
}
